app/controllers/api/v1/tag_followings_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TagFollowingsController < Api::V1::BaseController
+      before_action only: %i[index] do
+        require_access_token %w[read]
+      end
+
+      before_action only: %i[create destroy] do
+        require_access_token %w[read write]
+      end
+
+      rescue_from StandardError do
+        render json: I18n.t("api.endpoint_errors.tags.cant_process"), status: :bad_request
+      end
+
+      def index
+        render json: tag_followings_service.index.map(&:name)
+      end
+
+      def create
+        tag_followings_service.create(params.require(:name))
+        head :no_content
+      end
+
+      def destroy
+        tag_followings_service.destroy_by_name(params.require(:id))
```

Comment:
We can probably call the field "name" or something since that's what is being sent in.  I just went with the default off the path parser.  Let me look into it.

Reply:
Tell me when you take your decision, I will merge your PR after that.

Reply:
I'm not finding a way to override the name in Rails.  When I look at where I thought it was done it was an artifact of Rails providing different names a long a path so that a path of `posts/1234/comments/1` gets two parameters of posts_id and id.  If you want to squash and merge as is that works for me.